packages/ref-metrics-pmp/tests/unit/test-data/pdo-example/var_mode_PDO_EOF1_stat_cmip5_historical_mo_atm_ACCESS-ESM1-5_r1i1p1f1_2000-2005.json

```diff
@@ -0,0 +1,150 @@
+{
+    "REF": {
+        "obs": {
+            "defaultReference": {
+                "PDO": {
+                    "monthly": {
+                        "frac": 0.26088888494274287,
+                        "mean": -0.0045735968738200615,
+                        "mean_glo": 0.0848038963391425,
+                        "stdv_pc": 0.2359616756163025
+                    }
+                },
+                "period": "1900-2005",
+                "reference_eofs": 1,
+                "source": "/Users/jared/Library/Caches/pmp/v3.8.2/obs4MIPs_PCMDI_monthly/MOHC/HadISST-1-1/mon/ts/gn/v20210727/ts_mon_HadISST-1-1_PCMDI_gn_187001-201907.nc"
+            }
+        }
+    },
+    "RESULTS": {
+        "ACCESS-ESM1-5": {
+            "r1i1p1f1": {
+                "defaultReference": {
+                    "PDO": {
+                        "monthly": {
+                            "best_matching_model_eofs__cor": 1,
+                            "best_matching_model_eofs__rms": 1,
+                            "best_matching_model_eofs__tcor_cbf_vs_eof_pc": 1,
+                            "cbf": {
+                                "bias": -0.012902166924045364,
+                                "bias_glo": -0.027594450759993304,
+                                "cor": 0.797497520580893,
+                                "cor_glo": 0.539850032722944,
+                                "frac": 0.14558572272748813,
+                                "frac_cbf_regrid": NaN,
```

Comment:
@lee1043 FYI this isn't valid JSON. NaN isn't valid JSON it is implicitly converted to null, but some serialisers might explode. It might not be an issue in your tooling so far so it might be fine, but I'd like to covert it to null for the REF.